scripts/run.sh

```diff
@@ -210,14 +214,18 @@ ls -al ${LOG_TEMP}/${TASKNAME}.*
 
 # Data egress to directory
 if [ "${COPYRECO:-false}" == "true" ] ; then
-  # Token for write authentication
-  export BEARER_TOKEN=$(cat ${_CONDOR_CREDS}/eic.use)
-  if [ -n ${XRDWURL} ] ; then
-    xrdfs ${XRDWURL} mkdir -p ${XRDWBASE}/${RECO_DIR} || echo "Cannot write reconstructed outputs to xrootd server"
+  if [ "${USERUCIO:-false}" == "true" ] ; then
+    RUCIO_CONFIG=/opt/campaigns/hepmc3/scripts/rucio.cfg python /opt/campaigns/hepmc3/scripts/register_to_rucio.py -f "${RECO_TEMP}/${TASKNAME}.eicrecon.tree.edm4eic.root" -d "/${RECO_DIR}/${TASKNAME}.eicrecon.tree.edm4eic.root" -s epic
```

Comment:
Duplication of RUCIO_CONFIG setting, so I'd move it up top and make it available script-wide.